### PR TITLE
Improved Immovable Rod Targeting

### LINKED ISSUE #6467: Immovable rods miss too much
## Testmerges:
Huh. None.

## Reproduction:
1. Trigger rod event
2. Miss. Miss. Miss. Brush the AI sat slightly. Miss.
Might be just be a series of bad hits that I have seen but this has went on for long enough to be sus. It seems to, when it fires diagonally, eventually correct itself to fly straight up, which results in missing the station entirely.

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Immovable Rod now has a much higher chance to actually hit the station.
Closes #6467 

## Why It's Good For The Game
Whiffed rods are boring for everyone involved

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Tested with fifty rods, five missed.

</details>

## Changelog
:cl:
tweak: Immovable Rod is much more likely to hit the station
/:cl:

### CODE REVIEW

**Review by ike709 — CHANGES_REQUESTED** [review]
Literally just make a generator to bias it towards the center of the zlevel.

**Comment by ike709** [comment]
http://www.byond.com/docs/ref/#/proc/generator

**Comment by Ivniinvi** [comment]
Retested, 18 critical hits out of 20 rods fired post generatorification.

File: code/game/gamemodes/meteor/meteors.dm [hunk 1]
```diff
@@ -77,6 +77,45 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
...
+			startx = G.Rand()
+		if(WEST)
+			starty = G.Rand()
+			startx = (TRANSITIONEDGE+1)
+	. = locate(startx, starty, Z)
+
+/proc/aimbotDebrisFinishLoc(startSide, Z)
+	var/endy
+	var/endx
+	var/generator/G = generator("num", TRANSITIONEDGE + 1, world.maxx - (TRANSITIONEDGE + 1), "NORMAL_RAND")
```
Comment (ike709) [inline code/game/gamemodes/meteor/meteors.dm]: `NORMAL_RAND` is a macro, not a string.

Comment (ike709) [reply code/game/gamemodes/meteor/meteors.dm]:
```suggestion
	var/generator/G = generator("num", TRANSITIONEDGE + 1, world.maxx - (TRANSITIONEDGE + 1), NORMAL_RAND)
```

**Comment by imsynthex** [comment]
you should probably link this issue

- https://github.com/BeeStation/BeeStation-Hornet/issues/6467

**Review by ike709 — APPROVED** [review]

**Pull request merged**